Justfile

```
@@ -0,0 +1,7 @@
+build:
+    cargo bin trunk build
```

I believe we can use the `trunk` alias here

```suggestion
    cargo trunk build
```

---

That is true, in this specific case I just wanted to make it explicit that `cargo-run-bin` is being used to run `trunk` for someone who is reading the `Justfile` and maybe doesn't know the existence of the alias. I still think it would be best to stick with the more explicit `cargo bin trunk build` for this sake.